### PR TITLE
Separate groups and smart groups

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

File: go/contacts/templates/contacts/base.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content_breadcrumbs %}
-    <form id="search-filter" name="search-filter" action="" method="get" class="navbar-search pull-right">
+    <form id="search-filter"name="search-filter" action="" method="get" class="navbar-search pull-right">
```
Comment (justinvdm) [inline go/contacts/templates/contacts/base.html]: Intended change?

Comment (peterp) [inline go/contacts/templates/contacts/base.html]: yup, there was a space missing behind id and name.

Comment (justinvdm) [inline go/contacts/templates/contacts/base.html]: I'm confused, didn't we remove the space in this PR?

Comment (peterp) [inline go/contacts/templates/contacts/base.html]: No, no. I'm actually the one who's confused.

**Comment by justinvdm** [comment]
:+1: Once the unintended space change is gone (unless the change was intended and I'm just not understanding).

File: go/contacts/views.py [hunk 2]
```diff
@@ -70,7 +70,10 @@ def groups(request):
         for group_bunch in contact_store.groups.load_all_bunches(keys):
             groups.extend(group_bunch)
     else:
-        groups = contact_store.list_groups()
+        if type == 'smart':
+            groups = contact_store.list_smart_groups()
+        else:
+            groups = contact_store.list_groups()
```
Comment (hodgestar) [inline go/contacts/views.py:94]: I would have thought that if `type` equals `static` only static groups would be displayed?

Perhaps `type` should default to `None` which displays all groups and there should be an additional `if type == 'static':` branch in the `if / else`?

**Pull request merged**